automake 1.11 bails out with the following error:

Makefile.am:57: `INCLUDES' is the old name for `AM_CPPFLAGS' (or `*_CPPFLAGS')

And:

libtoolize: Consider adding `AC_CONFIG_MACRO_DIR([m4])' to configure.ac and
libtoolize: rerunning libtoolize, to keep the correct libtool macros in-tree.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 Makefile.am  |    2 +-
 configure.ac |    1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

Index: libmad-0.15.1b/Makefile.am
===================================================================
--- libmad-0.15.1b.orig/Makefile.am
+++ libmad-0.15.1b/Makefile.am
@@ -54,7 +54,7 @@ EXTRA_libmad_la_SOURCES =	imdct_l_arm.S 
 libmad_la_DEPENDENCIES =	@ASO_OBJS@
 libmad_la_LIBADD =		@ASO_OBJS@
 
-INCLUDES =		$(FPM) $(ASO)
+AM_CPPFLAGS =		$(FPM) $(ASO)
 
 BUILT_SOURCES =		mad.h
 CLEANFILES =		mad.h
Index: libmad-0.15.1b/configure.ac
===================================================================
--- libmad-0.15.1b.orig/configure.ac
+++ libmad-0.15.1b/configure.ac
@@ -25,6 +25,7 @@ AC_INIT([MPEG Audio Decoder], [0.15.1b],
 AC_PREREQ(2.53)
 
 AC_CONFIG_SRCDIR([decoder.h])
+AC_CONFIG_MACRO_DIR([m4])
 
 AM_INIT_AUTOMAKE
 
